modules/virt-verify-status-bootsource-update.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-verify-status-bootsource-update_{context}"]
+= Verify status of automatic updating for a boot source
+
+You can verify the status of a boot source to confirm that automatic importing and updating is enabled or disabled.
+
+The `status` section of each boot sources listed in the `dataImportChronTemplates` of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled. Examples of statuses include:
```

Comment:
There are two `dataImportChronTemplates` fields. One in the CR spec and one in the CR status. I think we need to be more explicit about it. Not sure about the exact wording. May be something like:
```suggestion
The `status` section of each boot sources listed in the `status.dataImportChronTemplates` of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled. Examples of statuses include:
```

or 

```suggestion
The `status` section of each boot sources listed in the `dataImportChronTemplates` of the `HyperConverged` CR's `status`, indicates if automatic importing and updating is enabled or disabled. Examples of statuses include:
```

Reply:
Yes, let's go with this:

_The `status` section of each boot sources listed in the `status.dataImportChronTemplates` field of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled._